tests/console/sudo.pm

```diff
@@ -31,11 +31,10 @@ sub sudo_with_pw {
     my $password = $args{password} //= $testapi::password;
     assert_script_run 'sudo -K';
     if ($command =~ /sudo -i|sudo -s|sudo su/) {
-        enter_cmd "expect -c 'spawn $command;expect \"password\" {send \"$password\\r\";interact'} default {exit 1}";
-        sleep 2;
+        enter_cmd "expect -c 'spawn $command;expect \"password for\" {send \"$password\\r\";interact'} default {exit 1}";
```

Comment:
The quoting here is actually broken:

```suggestion
        enter_cmd "expect -c 'spawn $command;expect \"password for\" {send \"$password\\r\";interact} default {exit 1}'";
```

Was this ever executed?

Reply:
included and verified